activesupport/lib/active_support/callbacks.rb

```diff
@@ -671,6 +671,7 @@ def default_terminator
               terminate
             end
           end
+          DEFAULT_TERMINATOR = DefaultTerminator.new.freeze
```

Comment:
Not sure why class is needed. Can just be `DEFAULT_TERMINATOR = Proc.new {...}`

Reply:
> we could use a singleton instance of a proc, but this should be slightly simpler for Ruby to execute